packages/plugins/__template/README.md

```diff
@@ -0,0 +1,32 @@
+# @blockly/plugin-template [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
```

Comment:
Thinking it might be nice to have some easy way to know all the things in the file that need to be updated. Maybe instead of @blockly/plugin-template -> @blockly/plugin-TODO or something obvious so people won't accidentally leave it?

Reply:
That sounds good.